inc/features/blocks/personalization/components/block-analytics.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Views from './views';
+
+const { useSelect } = wp.data;
+const { __ } = wp.i18n;
+
+const BlockAnalytics = ( { clientId } ) => {
+	const postId = useSelect( select => {
+		return select( 'core/editor' ).getCurrentPostId();
+	} );
+
+	// No post ID so post isn't published, don't show anything.
+	if ( ! postId ) {
+		return null;
+	}
+
+	// Fetch the stats.
+	const views = useSelect( select => {
+		return select( 'analytics/xbs' ).getViews( clientId, postId );
+	}, [ clientId, postId ] );
+	const isLoading = useSelect( select => {
+		return select( 'analytics/xbs' ).getIsLoading();
+	}, [ views ] );
```

Comment:
Does this actually depend on `views`?

Reply:
Hmm maybe not but it's only when views changes that it needs to be checked again.